### PR TITLE
Tempo: Add a placeholder to the TraceQL query editor

### LINKED ISSUE #55178: Tempo - Placeholder text in TraceQL editor
Let's add a placeholder text to the TraceQL editor, like what can be found for the TraceID input field. Monaco doesn't support an easy way to add placeholders, so let's find an alternative solution.

### PR DESCRIPTION
**Pull request by CrypticSignal**
This PR adds a `Enter a TraceQL query (run with Shift+Enter)` placeholder to the TraceQL query editor.

image

Closes [#55178](https://github.com/grafana/grafana/issues/55178).

### CODE REVIEW

**Review by adrapereira — COMMENTED** [review]
LGTM, but would like another review since we worked on this together and I might have missed something.

File: public/app/plugins/datasource/tempo/traceql/TraceQLEditor.tsx [hunk 1]
```diff
@@ -50,11 +53,40 @@ export function TraceQLEditor(props: Props) {
...
     />
   );
 }
 
+function setupPlaceholder(editor: monacoTypes.editor.IStandaloneCodeEditor, monaco: Monaco, styles: EditorStyles) {
+  const placeholderDecorators = [
+    {
+      range: new monaco.Range(1, 1, 1, 1),
+      options: {
+        className: styles.placeholder,
```
Comment (aocenas) [inline public/app/plugins/datasource/tempo/traceql/TraceQLEditor.tsx:67]: Let's just add a comment here that the placeholder content is in the CSS style itself so it's easier to understand this code.

**Review by aocenas — APPROVED** [review]
Just a small comment but otherwise looks good.

File: public/app/plugins/datasource/tempo/traceql/TraceQLEditor.tsx (hunk 1 above)
Comment (CrypticSignal) [reply public/app/plugins/datasource/tempo/traceql/TraceQLEditor.tsx]: [Done](https://github.com/grafana/grafana/pull/55257/commits/6148cf926c1302bc769e4315c9b61ae6f5ed23a6). Looks good?

Comment (aocenas) [reply public/app/plugins/datasource/tempo/traceql/TraceQLEditor.tsx]: Yep that works, thanks :)

**Pull request merged**